src/transport/fusedev/mod.rs

```diff
@@ -334,8 +334,8 @@ impl<'a, S: BitmapSlice> io::Write for FuseDevWriter<'a, S> {
 #[cfg(feature = "async-io")]
 mod async_io {
     use super::*;
-    use crate::transport::file_volatile_slice::FileVolatileBuf;
     use crate::transport::AsyncFileReadWriteVolatile;
+    use dbs_fuse::buf::FileVolatileBuf;
```

Comment:
Why is `FileVolatileBuf` moved?

Reply:
The unification is implemented by dbs-fuse, which is a unified wrapper for file io.
I will elaborate more on the commit message.